### PR TITLE
DCP: Mobile navigation menu expansion issue.

### PR DESCRIPTION
**Pull request by coderkakarrot**

### CODE REVIEW

File: docroot/themes/custom/dcp/sass/components/_menu.scss [hunk 1]
```diff
@@ -166,3 +166,79 @@ li a {
     margin-left: -45px;
   }
 }
+
+@media screen and (max-width: 768px) {
```
Comment (vidit-anjaria) [inline docroot/themes/custom/dcp/sass/components/_menu.scss]: @pratik17893 

The way media query written in sass that's not the sass way.. media query should be written component wise.

e.g.

.region-bottom {
  @media (max-width: 768px) {
       padding-top: 10px;
   }
   .container {
      @media (max-width: 768px) {
           font-size: 20px;
       }
    }
}

And moreover, we must be using breakpoint variable for tablet so better we use the same to maintain the code consistency.

**Review by vidit-anjaria — CHANGES_REQUESTED** [review]

**Review by vidit-anjaria — APPROVED** [review]
LGTM

File: docroot/themes/custom/dcp/templates/node/node--session.html.twig [hunk 2]
```diff
@@ -58,7 +58,7 @@
     </div>
   </div>
 </div>
-  {% if content.field_attachment %}
+  {% if content.field_attachment['#items'] %}
```
Comment (prashant114606-zz) [inline docroot/themes/custom/dcp/templates/node/node--session.html.twig]: @pratik17893 this is already updated in https://github.com/drupalpune/dcp/pull/59/files#diff-4724ad5cd18afb0d819e6556dc2f826eL61 and I have merged that PR first so can you please revert this piece of code

**Comment by prashant114606-zz** [comment]
LGTM!!
merging

**Pull request merged**